### PR TITLE
Creating the greatest vm ever :)

### PR DESCRIPTION
**Pull request by Alvin-He**
clearly I need somebody to code review as the c++ compiler is yelling at me every single time I trys to compile this :))))))))(

##############################
##############$###############
###########$$$$$$########$$####
$$$$$$$Afadsf#@%^&!#@$#########
###############################
^^^^my very sad attempt at making ascii art

### CODE REVIEW

**Comment by THRYLLR** [comment]
what



the FUCK



is this

File: benjamin.vm.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include sys.cpu
```
Comment (THRYLLR) [inline benjamin.vm.cpp:1]: incorrect syntax
idek if this is a library

File: benjamin.vm.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#include source
+#include memory
```
Comment (THRYLLR) [inline benjamin.vm.cpp:3]: same with these

File: benjamin.vm.cpp [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+int benjaminLength = 5ft;
```
Comment (THRYLLR) [inline benjamin.vm.cpp:5]: where'd you get the ft

File: benjamin.vm.cpp [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+memory* init(vm* env, string* sourceCode) {
```
Comment (THRYLLR) [inline benjamin.vm.cpp:7]: what are memories and vms

File: benjamin.vm.cpp [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  int length = sourceCode.length;
+  memory x69;
+  for (int i = 0; i < length; i++) {
+    if (sourceCode[i].subString(i, i + benjaminLength) == 'benjamin') {
+      x69 += memory::allocAndStore(cpu::calc(0x20*56-benjaminLength));
+    }else{
+      x69 += memory::allocAndStore(cpu::calc('emotional damage'.tonumber()));
```
Comment (THRYLLR) [inline benjamin.vm.cpp:14]: is this even cpp

File: benjamin.vm.cpp [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    }
+  }
+  return memory* (source::run(string *(&x69)));
+}
+
+int main() {
+  std::cout<< "https://www.youtube.com/watch?v=dQw4w9WgXcQ" << '\n';
+  std::cout << init(void* vm, souce::benjamin) << '\n';
```
Comment (THRYLLR) [inline benjamin.vm.cpp:22]: "souce"

**Review by THRYLLR — CHANGES_REQUESTED** [review]
🤣🤣🤣

File: benjamin.vm.cpp (hunk 1 above)
Comment (Alvin-He) [reply benjamin.vm.cpp:1]: do not worry about it 👍

File: benjamin.vm.cpp (hunk 3 above)
Comment (Alvin-He) [reply benjamin.vm.cpp:5]: it's called 'logic'

File: benjamin.vm.cpp (hunk 4 above)
Comment (Alvin-He) [reply benjamin.vm.cpp:7]: idk just left it there

File: benjamin.vm.cpp (hunk 6 above)
Comment (Alvin-He) [reply benjamin.vm.cpp:22]: that's what happens when you make a vm in 5 minutes

**Pull request merged**